File: platform/fastapi_server/computed_functions/async_example.compute.py
"""
Asynchronous compute function example.
"""

NAME = "async_example"


async def register(ctx):
    """Asynchronous function - can use await for I/O operations."""
    app_name = ctx.get("config", {}).get("app", {}).get("name", "unknown")
    return f"async_result_from_{app_name}"

File: platform/fastify_server/computed_functions/async_example.compute.mjs
/**
 * Asynchronous compute function example.
 */

export const NAME = "async_example";

export async function register(ctx) {
    // Asynchronous function - can use await for I/O operations
    const appName = ctx?.config?.app?.name || "unknown";
    return `async_result_from_${appName}`;
}
